test/LanguageServer.Engine.Tests/TaskScannerTests.cs

```diff
@@ -37,19 +37,19 @@ public TaskScannerTests(ITestOutputHelper testOutput)
                 Log.Information("Runtime Directory = {RuntimeDirectory}", RuntimeEnvironment.GetRuntimeDirectory());
 
                 Environment.SetEnvironmentVariable("MSBUILD_PROJECT_TOOLS_DOTNET_HOST_DIAGNOSTICS", "1");
-                RuntimeInfo = DotNetRuntimeInfo.GetCurrent(logger: Log);
+                RuntimeInfo = DotnetInfo.GetCurrent(logger: Log);
                 Environment.SetEnvironmentVariable("MSBUILD_PROJECT_TOOLS_DOTNET_HOST_DIAGNOSTICS", null);
             }
             else
-                RuntimeInfo = DotNetRuntimeInfo.GetCurrent();
+                RuntimeInfo = DotnetInfo.GetCurrent();
 
             Assert.NotNull(RuntimeInfo.BaseDirectory);
         }
 
         /// <summary>
         ///     Information about the current .NET runtime.
         /// </summary>
-        DotNetRuntimeInfo RuntimeInfo { get; }
+        DotnetInfo RuntimeInfo { get; }
```

Comment:
Probably a good idea to rename this property as well

Reply:
Good catch!